.github/workflows/build-and-deploy.yml

```diff
@@ -250,9 +250,20 @@ jobs:
           dir="$(cygpath -au artifacts)" &&
           mkdir -p "$dir" &&
 
+          case "$ARCHITECTURE" in
+            aarch64)
+              GIT_EXE="/clangarm64/bin/git.exe"
+              BUILD_SRC=
+              ;;
+            *)
+              GIT_EXE="/mingw64/bin/git.exe"
+              BUILD_SRC="YesPlease"
+              ;;
+          esac &&
```

Comment:
Shouldn't we have an `x86_64` arm and an `i686` arm in this `case` statement, and a `*)` arm that errors out?

Also, here would be a fine spot to define `MINGW_PREFIX`, probably instead of `GIT_EXE`.

Lastly, given how much time it takes to build the source package, maybe we should consider yet another "ARCHITECTURE" called `sources`?

Reply:
> The `build-and-deploy` pipeline is currently designed in a way that it uses a single `windows-latest` runner (`x86_64`) to build both `mingw32` and `mingw64` packages in one go. That's why I set up the logic this way.

Oh whoops. I totally was thinking about the `git-artifacts` workflow when I said this last night.

In light of that, even the `MINGW_PREFIX` suggestion does not make sense to me any longer, because there would have to be _two_ such prefixes in the non-ARM64 case.

So I retract my suggestions 😁